server/proxy/pf_channels.c

```diff
@@ -68,13 +77,28 @@ void pf_OnChannelConnectedEventHandler(void* data,
 	}
 	else if (strcmp(e->name, DISP_DVC_CHANNEL_NAME) == 0)
 	{
-		if (ps->disp->Open(ps->disp) != CHANNEL_RC_OK)
```

Comment:
Why this retry? I don't really understand your explanation, could you elaborate?

Reply:
yep.
first, i try to open disp. if it fails, i need to check if ERROR_NOT_FOUND is returned. it that's the case, then nothing should be done (client did not open disp). otherwise, it might not opened because server's dynvc wasn't in ready state yet. then, i wait for it to be in ready state and try again.

i must first check for ERROR_NOT_FOUND because drynvc will be in ready state only if a drdynvc caps sent from client.